packages/venia-concept/src/actions/user/api.js

```diff
@@ -0,0 +1,30 @@
+import { RestApi } from '@magento/peregrine';
+
+const { request } = RestApi.Magento2;
+
+export const getCountryName = (countries, countryId) => {
+    const country = countries.find(country => country.id === countryId);
+
+    return country ? country.full_name_locale : '';
```

Comment:
Do we really want to  return empty string here? What's the format for this data? Is there a safer fallback we can use or maybe just something like `COUNTRY_NAME_UNKNOWN`?